src/components/alerts/alerts.svelte

```diff
@@ -1,64 +1,41 @@
 <script>
-	import {
-		formatTimestamp,
-		validateTimestamp,
-		formatReasonType,
-		formatSeverityType
-	} from '$lib/formatters';
+	import { formatTimestamp, validateTimestamp } from '$lib/formatters';
 
 	let { situations = [] } = $props();
 
 	const situation = situations[0] ?? {};
 	const activeWindow = situation.activeWindows?.[0] ?? {};
 
 	const title = $derived(situation.summary?.value ?? '');
-	const description = $derived(situation.description?.value ?? '');
 
 	const dateStart = formatTimestamp(activeWindow.from ?? '');
 	const dateEnd = formatTimestamp(activeWindow.to ?? '');
 
-	const reason = formatReasonType(situation.reason);
-	const severity = formatSeverityType(situation.severity);
-
-	const gray_bg = 'bg-[rgba(0,39,59,0.08)] font-bold px-5.5 py-1.5 corner rounded-[10px]';
+	const gray_bg = 'bg-[rgba(0,39,59,0.05)] font-bold px-5.5 py-1.5 corner rounded-[24px]';
```

Comment:
this would be a good constant to extract into a semantically named css class (or even better, a semantically named new Tailwind color!)